core/domain/feedback_services_test.py

```diff
@@ -87,11 +86,13 @@ class SuggestionQueriesUnitTests(test_utils.GenericTestBase):
     def _generate_thread_id(self, unused_exp_id):
         return self.THREAD_ID1
 
-    def _get_threads(self, unused_exploration_id):
-        return self.threads
+    def _put_threads(self):
```

Comment:
And maybe move test_create_and_get_suggestion() to a new suite or bundle it with one of the other suites -- or alternatively modify it so that it doesn't interfere with what's already been done in setUp().